animations/lib/main.dart

```diff
@@ -4,6 +4,7 @@
 
 import 'dart:io';
 
+import 'package:animations/src/misc/flutter_animate.dart';
```

Comment:
Seems you use local imports for the other demos.

Reply:
Whups, I forgot to add it to the `misc.dart` export list. Fixed.